crates/uv/src/commands/pip_install.rs

```diff
@@ -389,6 +391,21 @@ async fn build_editables(
         .into_iter()
         .collect();
 
+    // Validate that the editables are compatible with the target Python version.
```

Comment:
Not that I know anything about `uv` codebase but FTR this fix should also be applied on non-editable local installs. (appologies if that's already done).

Thanks for working on this!

Reply:
I thought we already enforced those, but I guess not! Will do it in a separate PR, thanks.